package.json

```diff
@@ -32,29 +32,29 @@
   "dependencies": {
     "@types/minimatch": "3.0.3",
     "fs-extra": "^8.1.0",
-    "handlebars": "^4.4.3",
-    "highlight.js": "^9.15.10",
+    "handlebars": "^4.5.1",
+    "highlight.js": "^9.16.2",
     "lodash": "^4.17.15",
     "marked": "^0.7.0",
     "minimatch": "^3.0.0",
     "progress": "^2.0.3",
     "shelljs": "^0.8.3",
-    "typedoc-default-themes": "^0.6.0",
-    "typescript": "3.6.x"
+    "@gerrit0/typedoc-default-themes": "^0.6.4",
```

Comment:
Can this be reverted once you publish to the main namespace?

Reply:
Yes, I kept it here for now so that tests would keep passing. It should be reverted before a new theme version is published.

Reply:
It has now been reverted